src/actions/firestore.js

```diff
@@ -230,16 +230,11 @@ export function setListener(firebase, dispatch, queryOpts, successCb, errorCb) {
           : docData.docChanges;
       // Dispatch different actions for doc changes (only update doc(s) by key)
       if (docChanges && docChanges.length < docData.size) {
-        if (docChanges.length === 1) {
```

Comment:
There is no need to check for the length of docChanges.  Looping through the array will do the trick.